tests/datatypes/test_bytes.py

```diff
@@ -20,6 +20,20 @@ def test_getattr(self):
                 print(err)
             """)
 
+    def test_capitalize(self):
+        self.assertCodeExecution("""
+            print(b'hello, world'.capitalize())
+            print(b'helloWORLD'.capitalize())
+            print(b'HELLO WORLD'.capitalize())
+            print(b'2015638687'.capitalize())
```

Comment:
Looking at #530 - I think the fix is trivial, so it may be worth rolling it into this PR.